packages/react-reconciler/src/__tests__/ReactHooks-test.internal.js

```diff
@@ -347,14 +348,17 @@ describe('ReactHooks', () => {
       });
       return <Child text={text} />;
     }
+    let root;
+    act(() => {
+      root = ReactTestRenderer.create(null, {unstable_isConcurrent: true});
```

Comment:
Nit: If you move the `ReactTestRenderer.create` call outside `act`, does it warn? Maybe we can special case `null`? (I don't like how test renderer's `create` also updates at the same time, and I'm accustomed to passing null as a workaround.)

Reply:
It does not warn when you pass null, we only warn on update calls. It will warn if you try to render something with an effect ofc  (once https://github.com/facebook/react/pull/15631 lands) To clarify, this is the behaviour you want, yeah? I'll move it out in this test.